torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -496,18 +502,23 @@ static Tensor _make_dual(
     return at::alias(primal);
   })();
   std::function<at::Tensor(const at::Tensor&)> func = nullptr;
+  std::function<at::Tensor(const at::Tensor&)> rev_func = nullptr;
   if (!primal.unsafeGetTensorImpl()->support_as_strided()) {
     auto size_vec = primal.sizes().vec();
     func = [=](const at::Tensor& input_base) {
       return input_base.view(size_vec);
     };
+    rev_func = [=](const at::Tensor& input_view) {
+      return input_view.view(size_vec);
```

Comment:
Hmm I don't think these actually follow the invariant of rev_view(view(base)) having the same metadata as base. Maybe just put a error for now saying its not supported for now?

Reply:
Added an error here and below